src/librustc_mir/borrow_check/nll/mod.rs

```diff
@@ -392,7 +393,7 @@ impl ToRegionVid for RegionVid {
     }
 }
 
-fn live_variable_set(regular: &LocalSet, drops: &LocalSet) -> String {
+fn live_variable_set<V: Idx>(regular: &LocalSet<V>, drops: &LocalSet<V>) -> String {
```

Comment:
Similarly, for this first refactoring, we would have `LocalSet<Local>` here -- the idea is that the **liveness computation** is generic over the index, but we **as the users of it** still want to compute the liveness of **every** MIR local variable to start